### PR TITLE
Fix nullable externalMeetingId

### PR DESCRIPTION
**Pull request by hokyungh**
### Description of changes:
According to https://docs.aws.amazon.com/chime/latest/APIReference/API_Meeting.html, externalMeetingId is optional field.
### Testing done:
Deploy server that calls `createMeeting` without externalMeetingId and verify that screen share and other functionalities work.
Also tested with current server that there is no behavior difference.

#### Manual test cases (add more as needed):

- [X] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [X] Leave meeting
- [ ] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [ ] Send audio
- [ ] Receive audio
- [ ] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -14,17 +15,17 @@
             <objects>
                 <viewController storyboardIdentifier="main" id="BYZ-38-t0r" customClass="JoiningViewController" customModule="AmazonChimeSDKDemo" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="8bC-Xf-vdC">
-                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
```
Comment (zhinang-amazon) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/Base.lproj/Main.storyboard:18]: Revert the change in this storyboard file?

**Review by zhinang-amazon — APPROVED** [review]

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift [hunk 2]
```diff
@@ -18,7 +18,7 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
     @IBOutlet var joinWithoutCallKitButton: UIButton!
     @IBOutlet var joinAsIncomingCallButton: UIButton!
     @IBOutlet var joinAsOutgoingCallButton: UIButton!
-    @IBOutlet var DebugSettingsButton: UIButton!
+    @IBOutlet var debugSettingsButton: UIButton!
```
Comment (ziyiz-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift:21]: thx!

**Review by ziyiz-amzn — APPROVED** [review]
